docs/api-reference.md

```diff
@@ -77,6 +77,8 @@ Similar to connection-builder but is used to build the JDBC properties file. For
 
 Defines the unique set of connection attributes which is used to generate a connection "key" and has important security considerations. Connections can be reused and shared within Tableau processes based on this key, so it must contain attributes whose values will be unique in a given security context. Username is a commonly used attribute that will make a unique connection for each user, for example.
 
+Starting in Tableau 2021.1 a connector using Connection Dialog V2 style, connection-fields, can let the system determine at runtime the correct connection-normalizer by not defining it. Any connection-normalizer defined in the tdr file will take precedence at runtime. To use set min-version-tableau='2021.1' or newer in the manifest file.
```

Comment:
The connector packager will overwrite this I'm pretty sure. We can either update the packager to not overwrite a higher min-tableau-version than it calculates, or to correctly stamp the min-tableau-version when using the inferred resolver.

Reply:
Tracking both in 1232944 and 1232398.

Reply:
👍